bucket.go

```diff
@@ -20,6 +20,8 @@ type Bucket interface {
 	GetName() string
 	Get(k string, rv interface{}) error
 	GetRaw(k string) ([]byte, error)
+	Gets(k string, rv interface{}, caso *uint64) error
```

Comment:
Brought that over from go-couchbase's Gets, but agree it's not needed here.  Will switch to cas.